price-feeder/oracle/oracle.go

```diff
@@ -287,6 +287,13 @@ func (o *Oracle) getOrSetProvider(providerName string) provider.Provider {
 		case config.ProviderHuobi:
 			priceProvider = provider.NewHuobiProvider()
 
+		case config.ProviderOkx:
+			pp, err := provider.NewOkxProvider(context.TODO(), o.providerPairs[config.ProviderOkx]...)
+			if err != nil {
+				fmt.Printf("ProviderOkx error %+v", err)
+			}
+			priceProvider = pp
```

Comment:
The method that calls `getOrSetProvider` should probably error out if okx errors out

```suggestion
		case config.ProviderOkx:
			priceProvider, err := provider.NewOkxProvider(context.TODO(), o.providerPairs[config.ProviderOkx]...)
			if err != nil {
				provider.Provider{}, err
			}
```
Also, logs within `oracle.go` should be done with the oracle logger, e.g.,: 
```golang
o.logger.Err(err).Msg("oracle tick failed")
```